src/clusterfuzz/_internal/datastore/data_types.py

```diff
@@ -423,6 +423,9 @@ class Testcase(Model):
   # Timestamp.
   timestamp = ndb.DateTimeProperty()
 
+  # Original creation timestamp.
```

Comment:
Can we add some comments to the existing `timestamp` property above? To make it clear how this is different? 

I think something like

"# The time when this Testcase is considered valid. This is usually when the Testcase is created in Datastore when created in e.g. `fuzz_task`. However, there are also cases where the `Testcase` is a placeholder (e.g. awaiting `analyze`), which means this timestamp will reflect when `analyze` completes its analysis).